mod/cp_notifications/start.php

```diff
@@ -1212,18 +1285,21 @@ function cp_create_notification($event, $type, $object) {
 
 			$to_recipients = get_subscribers($dbprefix, $author_id, $content_id);
 			$to_recipients_site = get_site_subscribers($dbprefix, $author_id, $content_id);
-			$email_only = false;
 			break;
 
 	} // end of switch statement
 
 
+
+    $notification_error_type = "";
```

Comment:
Not sure if this was another test piece, but it also needs an indent

Reply:
weird, the indentation seems fine on my sublime

![image](https://user-images.githubusercontent.com/5798645/29889164-3c247ee6-8d91-11e7-848b-8df054c942bc.png)

Reply:
This line is using spaces, the rest are tabs. https://stackoverflow.com/questions/10153998/sublime-text-2-view-whitespace-characters
Also would be fixed if the EditorConfig PR lands

Reply:
changed the spaces to indent